dev-python/factory-boy/factory-boy-3.2.1.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{9..11} )
+
+inherit distutils-r1 pypi
+
+DESCRIPTION="A test fixtures replacement for Python"
+HOMEPAGE="
+	https://github.com/FactoryBoy/factory_boy/
+	https://factoryboy.readthedocs.io/
+	https://pypi.org/project/factory-boy/
+"
+MY_PN="factory_boy"
+SRC_URI="$(pypi_sdist_url)"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+
+DEPEND="
+	test? (
+		dev-python/django[${PYTHON_USEDEP}]
+		dev-python/sqlalchemy[sqlite,${PYTHON_USEDEP}]
+		dev-python/flask-sqlalchemy[${PYTHON_USEDEP}]
+		dev-python/mongoengine[${PYTHON_USEDEP}]
+		dev-python/pymongo[${PYTHON_USEDEP}]
+	)
+"
+
+S="${WORKDIR}/${MY_PN}-${PV}"
```

Comment:
The eclass also sets `S` accordingly, so you can remove this and `MY_PN` as well.